code/game/jobs/job/security.dm

```diff
@@ -259,4 +259,39 @@
 	backpack = /obj/item/storage/backpack/security
 	satchel = /obj/item/storage/backpack/satchel_sec
 	dufflebag = /obj/item/storage/backpack/duffel/security
-	box = /obj/item/storage/box/engineer
\ No newline at end of file
+	box = /obj/item/storage/box/engineer
+
+
+/datum/job/secinstructor
+	title = "Drill Instructor"
+	flag = INSTRUCTOR
+	department_flag = KARMA
+	total_positions = 1
+	spawn_positions = 1
+	is_security = 1
+	supervisors = "the head of security"
+	department_head = list("Head of Security")
+	selection_color = "#ffeeee"
+	access = list(access_security, access_sec_doors, access_brig, access_court, access_maint_tunnels, access_morgue, access_weapons)
```

Comment:
Why do they have morgue access? Pretty sure Sec Officers don't, so a Sec Instructor shouldn't need it either.

Reply:
/shrug/
Im just doing as I was told

Reply:
Irrespective of my opinions on this PR, I can't see morgue access as being necessary, can you?